### DISCUSSION TITLE
Custodian

### DISCUSSION THREAD

**Discussion by cesar-sanjuan** [globaleaks/globaleaks-whistleblowing-software#3432] [General]
Hello
I can't find documentation on the properties or functions that a user of the keeper type has.
Can you tell me what function it has or where to look?
I'm sorry for the inconvenience
Thank you

**Comment by evilaliv3** [comment]
Thank you  @cesar-sanjuan 

Actually this feature is a feature that we have implemented to support a specific project run by the italian national authority and for this reason we have not found useful to document it widely.

The custodian user role is a role that can authorize a recipient to access the whistleblower identity (if provided by the whistleblower) 
This user have the only responsibility to authorize or deny the access on the base of a motivation provided by the recipient when making the request, but has no access to the report or to the identity.

**Reply by cesar-sanjuan** [reply]
good morning
thanks for your answer
And for her a new doubt arises. They say that it allows access to the identity of the complainant but we have not found in any parameter how to activate the fields for the identity of the complainant and in the default forms we see that the complaints are anonymous. Is it possible to activate fields for the identity to be displayed?

**Comment by evilaliv3** [comment]
If you look at the feature "Questionnaire template" you should find a template dedicated to optionally collecting the whistleblowing identity if whistleblowers wants or have the need to declare themeselves.

**Reply by mbolzan** [reply]
Hi Giovanni,

Edited: found answer to first question: let see https://github.com/orgs/globaleaks/discussions/3497#discussioncomment-6657931
...and if the list of question types does not include an 'identity' type? have you a template to import?


where is the possibility from the recipient to send a request to the custodian?
we enable (from admin) the feature Custodian, create a user but he see olny an empty list of requests.
